README.md

```diff
@@ -1,21 +1,20 @@
 # \<spine-button\>
 
-A wrapper around the `paper-button` providing additional mixins and variables for styling.
+An element that inherits the `paper-button` behavior and appearance
```

Comment:
inherits the `paper-button`__**'s**__ behavior

Having `'s` seems to be more correct here. Please review all other occurrences of this phrase in the other files' docs.